--- __backup0	2022-10-06 05:32:48.011761290 +0000
+++ src/BranchChecker_avl.c	2022-10-06 05:34:54.817898557 +0000
@@ -215,5 +215,5 @@
         if (node->Left == NULL && node->Right == NULL) {
             node = NULL;
         }
-        else if (node->Left != NULL) { //Original: node->Left != NULL && node->Right == NULL
+        else if (!(node->Right > 0) && (node->Left != ((void *)0))) {
             node->Left->Parent = node->Parent;
